var/spack/repos/builtin/packages/mpas-model/package.py

```diff
@@ -72,6 +81,7 @@ def target(self, model, action):
         return targets
 
     def build(self, spec, prefix):
+        copy_tree('MPAS-Data/atmosphere', 'src/core_atmosphere/physics')
```

Comment:
```suggestion
        copy_tree(join_path('MPAS-Data', 'atmosphere')),
                  join_path('src', 'core_atmosphere', 'physics'))
```
Please avoid / as directory delimiter for non-unix platform.

Reply:
I've fixed it. Thank you for your help.